tests/README.md

```diff
@@ -1,13 +0,0 @@
-### How to run tests
```

Comment:
Please add this back, I find it helpful to keep the test instructions in the test folder.

Reply:
Added back. I've also added a description for some useful commands.